Documentation/cross-building.md

```diff
@@ -107,7 +107,9 @@ make -j8 objwriter
 
 7. And to execute use:
 ```
-./corerun ilc.dll --codegenopt "AltJitNgen=* AltJit=*" --verbose @Hello.ilc.rsp
+./corerun ilc.dll --codegenopt AltJitNgen=* --verbose @Hello.ilc.rsp
```

Comment:
Should `AltJitNgen=*` still be in quotes so that the `*` wildcard is not accidentally expanded by the shell?

Reply:
Both versions work. Quotes are optional.

Reply:
If you happen to have `AltJitNgen=foo` file on disk in your current directory, the version without quotes is not going to work.

It is unlikely that you will `AltJitNgen=foo` file on disk in your current directory. But it is a very good habit to always enclose anything with `*` that is not meant to be expanded in quotes.